auth-tokens/src/test/java/com/palantir/tokens/auth/UnverifiedJsonWebTokenTests.java

```diff
@@ -26,83 +26,101 @@
 
 final class UnverifiedJsonWebTokenTests {
 
-    private static final BearerToken ALL_CLAIMS_TOKEN = BearerToken.valueOf("header."
-            + "eyJzdWIiOiJ3NVAyV1FNQlEwNnB5WEl3U2xCLy9BPT0iLCJzaWQiOiJQOFpqMUQ1SVRlMjZUdGVLK1l1RFl3PT0iLCJqdGkiOiJwRm0w"
-            + "b1ZDSlQrQ0dWZFhmMmJLMy9RPT0iLCJvcmciOiJGQlMycTgvbFQvMnNBRktxZ09pUW13PT0iLCJleHAiOiAxNTc3ODY1NjAwfQ"
-            + ".signature");
-
-    private static final BearerToken REQUIRED_CLAIMS_TOKEN = BearerToken.valueOf(
-            "header." + "eyJzdWIiOiJ3NVAyV1FNQlEwNnB5WEl3U2xCLy9BPT0iLCJleHAiOiAxNTc3ODY1NjAwfQ" + ".signature");
-
-    private static final BearerToken INVALID_BEARER_TOKEN = BearerToken.valueOf("InvalidBearerToken");
-
-    private static final BearerToken INVALID_ENCODING_TOKEN = BearerToken.valueOf("header."
-            + "eyJzdWIiOiJrazlVMHB0ZVJ3K1FYYk55ZkZkcklBPT0iLCJqdGkiOiJ2MEtCNWdVTFJkT3dFWWh4Z1o3bERnPT0ifQo+"
-            + ".signature");
-
-    private static final BearerToken INVALID_PAYLOAD_TOKEN = BearerToken.valueOf("header."
-            + "eyJzdWIiOiJrazlVMHB0ZVJ3K1FYYk55ZkZkcklBPT0iLCJqdGkiOiJ2MEtCNWdVTFJkT3dFWWh4Z1o3bERnPT0iCg"
-            + ".signature");
+    private static final BearerToken ALL_CLAIMS_TOKEN = BearerToken.valueOf("""
+        header.\
+        eyJzdWIiOiJ3NVAyV1FNQlEwNnB5WEl3U2xCLy9BPT0iLCJzaWQiOiJQOFpqMUQ1SVRlMjZUdGVLK1l1RFl3PT0iLCJqdGkiOiJwRm0wb1ZDSlQ\
+        rQ0dWZFhmMmJLMy9RPT0iLCJvcmciOiJGQlMycTgvbFQvMnNBRktxZ09pUW13PT0iLCJzdmMiOiJzZXJ2aWNlIn0\
```

Comment:
(not necessarily for this PR - more of a meta-comment) Do you think it would be reasonable to avoid having the base64-encoded tokens as the hardcoded value here, and instead have something like a util method that takes the decoded base64 value (e.g. `{"sub":"w5P2WQMBQ06pyXIwSlB//A==","sid":"P8Zj1D5ITe26TteK+YuDYw==","jti":"pFm0oVCJT+CGVdXf2bK3/Q==","org":"FBS2q8/lT/2sAFKqgOiQmw==","svc":"service"}` here) and build the token out of it? That would make it slightly easier to review and understand what to expect in the future 🤔